RockLib.Messaging.SQS/SQSReceiver.cs

```diff
@@ -279,30 +283,34 @@ private async Task ReceiveMessages()
                             break;
                         }
                     }
-                    catch (Exception ex)
+                    catch (OverLimitException ex)
```

Comment:
As much as I want us to have better exception handling, in cases like this....are other exceptions going to leak out? I'd consider adding this to the changelog depending on where `ReceiveMessages()` is called.

Reply:
I changed it back to catching general exceptions and adding a `#pragma warning disable` to it.  That's what I did in other places in that same file, so I'm not sure why I changed just this one lol.